stories/src/helpers.tsx

```diff
@@ -221,25 +219,25 @@ const findFields = (table: Table) => {
   return fields
 }
 
-export const lattitudeKnob = (table: Table, initial?: Record<string, any>) => {
+export const lattitudeKnob = (table: Table) => {
   return select(
     'lattitude',
     {
       ...findFields(table),
       ...findTags(table, true),
     },
-    initial || '_value'
+    '_value'
   )
 }
 
-export const longitudeKnob = (table: Table, initial?: Record<string, any>) => {
+export const longitudeKnob = (table: Table) => {
   return select(
     'longitude',
     {
       ...findFields(table),
       ...findTags(table, true),
     },
-    initial || '_value'
+    '_value'
```

Comment:
none of the places these functions were called made use of the second argument. I removed it due to TypeScript having problems with types.